demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py

```diff
@@ -48,6 +46,26 @@ class XsoarClient(BaseModel):
     class Config:
         arbitrary_types_allowed = True
 
+    @classmethod
+    def is_xsoar_healthy(cls, client: DefaultApi) -> bool:
```

Comment:
is_server_healthy and lets use inheritance

Reply:
called it `is_healthy`

Reply:
done